### ISSUE TITLE
Military Tools Analysis Ribbon does not launch tools from Conda package if installed

### ISSUE THREAD

**Issue by csmoore** [Esri/military-tools-desktop-addins#97]
While [diagnosing another issue](https://github.com/ArcGIS/solutions-defense/issues/946) - it was noticed that the conda/system installed version of MilitaryTools toolbox was not being executed when the conda package is installed.

I believe the intended behavior was to first try to execute the system/conda tool if installed and then fall back to a local copy installed to the addin folder if the system/conda toolbox was not available.

The problem is here: 
https://github.com/Esri/military-tools-desktop-addins/blob/dev/source/MilitaryToolsForArcGISPro/MilitaryToolsModule.cs#L54 

Just wanted to check with anyone who might know why this was removed (like @saip ) just to make sure removing this capability wasn't intentional (because of a problem) before I put it back in. 

There really isn't a user-repro case for this, you would need to 1) install the conda package and 2) delete the toolbox folder from the addin install location to test this.

**Comment by csmoore** [comment]
One case where using the Conda package first might be undesirable is if the user has an older version installed. Ex. conda package 3.0 is installed and latest version is 3.3

**Comment by BobBooth** [comment]
@csmoore - good point.  I do not know why that functionality was changed/removed.
Is it possible to determine if & what version of the conda package is installed when installing the add-in?
Might be something we'd look at later.

**Comment by lfunkhouser** [comment]
Believe this is resolved. Needs to be verified.

**Comment by csmoore** [comment]
I confirmed that the tools will run from the Conda package if the tools in the local cache folder are not present:

1. Run ArcGIS Pro (Pro must be running first)
2. Go to 
%LOCALAPPDATA%\ESRI\ArcGISPro\AssemblyCache\{bea7f059-c69e-4e17-84bd-072236629c0c}
3. Rename : "toolboxes" to "toolboxes-RENAMED"
4. Verify that GP tools run from Military Tools button work

![image](https://user-images.githubusercontent.com/3090809/59712903-2e789d80-91dc-11e9-840e-c075ed0d41f0.png)

Closing issue
